src/main/java/com/flippingutilities/ui/MasterPanel.java

```diff
@@ -114,55 +103,49 @@ public MasterPanel(FlippingPlugin plugin,
+	private JPanel createHeader(JComboBox accountSelector, MaterialTabGroup tabSelector)
 	{
-		settingsButton.setBorder(new EmptyBorder(0,0,2,0));
-
 		JPanel accountSelectorPanel = new JPanel(new BorderLayout());
-		accountSelectorPanel.setBackground(ColorScheme.DARKER_GRAY_COLOR.darker());
+		accountSelectorPanel.setBackground(CustomColors.DARK_GRAY);
 		accountSelectorPanel.add(accountSelector, BorderLayout.CENTER);
 		accountSelectorPanel.setBorder(new EmptyBorder(0,0,4,0));
 
-		JPanel tabGroupArea = new JPanel(new BorderLayout());
-		tabGroupArea.setBackground(ColorScheme.DARKER_GRAY_COLOR.darker());
-		tabGroupArea.add(tabSelector, BorderLayout.CENTER);
-		tabGroupArea.add(settingsButton, BorderLayout.EAST);
-		tabGroupArea.add(communityPanel(), BorderLayout.NORTH);
-
 		JPanel header = new JPanel(new BorderLayout());
-		header.setBackground(ColorScheme.DARKER_GRAY_COLOR.darker());
+		header.setBackground(CustomColors.DARK_GRAY);
 		header.add(accountSelectorPanel, BorderLayout.NORTH);
-		header.add(tabGroupArea, BorderLayout.CENTER);
-
+		header.add(createCommunityPanel(), BorderLayout.CENTER);
+		header.add(tabSelector, BorderLayout.SOUTH);
+		header.setBorder(new EmptyBorder(0,0,3,0));
 		return header;
 	}
 
-	private JPanel communityPanel() {
+	private JPanel createCommunityPanel() {
 		JPanel communityPanel = new JPanel(new BorderLayout());
-		communityPanel.setBackground(ColorScheme.DARKER_GRAY_COLOR.darker());
-
-		JPanel topPanel = new JPanel();
-		topPanel.setBackground(ColorScheme.DARKER_GRAY_COLOR.darker());
-
-		JLabel topLabel = new JLabel("Join discord for bot dump alerts!", JLabel.CENTER);
-		topLabel.setForeground(CustomColors.VIBRANT_YELLOW);
-		topLabel.setFont(FontManager.getRunescapeSmallFont());
-		topLabel.setBackground(ColorScheme.DARKER_GRAY_COLOR.darker());
-
-		JLabel questionMarkLabel = new JLabel(Icons.QUESTION_MARK);
-		JPopupMenu popup = new JPopupMenu();
-		//popup.setPreferredSize(new Dimension(100,100));
-		popup.add(new JLabel(Icons.DUMP_ALERT_PIC));
-		UIUtilities.addPopupOnHover(questionMarkLabel, popup, false);
-
-		topPanel.add(topLabel);
-		topPanel.add(questionMarkLabel);
+		communityPanel.setBackground(CustomColors.DARK_GRAY);
+		communityPanel.setBorder(new EmptyBorder(7,0,0,0));
+
+//		JPanel topPanel = new JPanel();
+//		topPanel.setBackground(CustomColors.DARK_GRAY);
+//		topPanel.setBorder(new EmptyBorder(0,0,10,0));
+//
+//		JLabel topLabel = new JLabel("Join discord for bot dump alerts!", JLabel.CENTER);
+//		topLabel.setForeground(ColorScheme.GRAND_EXCHANGE_PRICE);
+//		topLabel.setFont(FontManager.getRunescapeSmallFont());
+//		topLabel.setBackground(CustomColors.DARK_GRAY);
+//
+//		JLabel questionMarkLabel = new JLabel(Icons.QUESTION_MARK);
+//		JPopupMenu popup = new JPopupMenu();
+//		popup.add(new JLabel(Icons.DUMP_ALERT_PIC));
+//		UIUtilities.addPopupOnHover(questionMarkLabel, popup, false);
+//
+//		topPanel.add(topLabel);
+//		topPanel.add(questionMarkLabel);
```

Comment:
To delete?

Reply:
we can delete it, left it in there cause its probably not gonna be too long before we have some sort of banner text/something in its place.